commit 8d5e749592e623cfb4cbacbcfa6674528a81fe41 (refs/changes/74/474/1)
Author: David Pacheco <dap@joyent.com>
Date:   2016-09-15T13:32:33-07:00 (3 years, 1 month ago)
    
    TOOLS-1546 cr.joyent.us "new user guide" link overlaps with header on change screen

diff --git a/images/appserver/theme/GerritSite.css b/images/appserver/theme/GerritSite.css
index 63dfb71..6be053c 100644
--- a/images/appserver/theme/GerritSite.css
+++ b/images/appserver/theme/GerritSite.css
@@ -2,10 +2,10 @@
  * See the Gerrit documentation for information about this file.
  */
 table.crjHeader {
-	width: 100%;
 	position: fixed;
 	right: 5px;
-	top: 35px;
+	top: 27px;
+	width: 300px;
 
 	font-weight: bold;
 	text-align: right;
